apiBase/apiRequest.js

```diff
@@ -20,9 +20,13 @@ export default (apiOptions, method, path, options={}) => {
     ? { ...defaultHeaders, Authorization: `Bearer ${token}` }
     : defaultHeaders;
   const _path = path.startsWith('/') ? path : `/${path}`
-  const _query = { ...query, app: `${appName}-${env}` };
   const endpoint = origin + _path;
 
+  let _query = { ...query, app: `${appName}-${env}` };
+  if (_method === 'get') {
+    _query = { ...query, raw_json: 1 };
```

Comment:
Just to put this in stone, as per our conversation, please do not merge this until `raw_json` is removed. It doesn't belong in `apiRequest` as this is agnostic to our endpoints. We're making the assumption that this should apply to every endpoint currently and going forward. Additionally, if we ever want to remove it, it becomes a very scary prospect. Finally, if we ever want to request something without `raw_json`, we won't be able to.